docs/how-to/quantization.md

```diff
@@ -137,31 +87,60 @@ quantized_model = quantize_qat(model_fp32, model_quant)
-```
-Please see [E2E_example_model/e2e_user_yolov3_example.py](https://github.com/microsoft/onnxruntime/blob/master/onnxruntime/python/tools/quantization/E2E_example_model/object_detection/trt/yolov3/e2e_user_yolov3_example.py) for more details.
-### Evaluation for quantization
-#### COCO dataset evaluation
-This tool integrates the COCO dataset API to evaluate model prediction. Please make sure to install COCO API first (```pip install pycocotools```).
-#### Example
-{: .no_toc }
-```
-dr = YoloV3DataReader(validation_dataset, model_path=model_path, start_index=i, size_limit=stride, batch_size=20, is_evaluation=True)
-evaluator = YoloV3Evaluator(model_path, dr, providers=providers)
-evaluator.predict()
-results += evaluator.get_result()
-...
-evaluator.evaluate(results, annotations)
-```
-Please see [E2E_example_model/e2e_user_yolov3_example.py](https://github.com/microsoft/onnxruntime/blob/master/onnxruntime/python/tools/quantization/E2E_example_model/object_detection/trt/yolov3/e2e_user_yolov3_example.py) for more details.
+## Which quantization data type to use?
+Quantization represents value with 8 bit, which can be either int8 and uint8. Combining with activation and weight, the data format can be (activation:uint8, weight:uint8), (activation:uint8, weight:int8), etc.
+
+Let's use U8U8 as as shorthand for (activation:uint8, weight:uint8), and U8S8 for (activation:uint8, weight:int8), and S8U8, S8S8 for other two formats.
+
+Currently, OnnxRuntime CPU only supports activation with type uint8, i.e., U8X8 only.
+
+### x86-64
+#### AVX2
+Try U8U8 firstly, and then U8S8.
+- Performance
+U8S8 leverage VPMADDUBSW instruction but U8U8 kernel can process 6 rows at a time versus 4 rows for the U8S8 kernel, and U8U8 sequence is two instructions vs three instructions for U8S8. Thus, this balance ends up with u8u8 just in reach of U8S8 for older HW (Broadwell)
+- Accuracy
+VPMADDUBSW has saturation issue, thus U8S8 needs to use [reduce_range](https://github.com/microsoft/onnxruntime/blob/master/onnxruntime/python/tools/quantization/quantize.py) if accuracy is not good enough.
+
+#### AVX512
+U8S8 is much faster, but may suffer saturation issue.
+- Performance
+U8S8 can only do twice mul/adds per instruction as much as U8U8. Typically U8S8 sequence ends up winning, can be twice as fast as u8u8.
+- Accuracy
+Same as the AVX2 because of VPMADDUBSW. Needs to ure reduce_range if accuracy is not good enough.
+
+#### VNNI
+No difference.
+
+### ARM64
+U8S8 can be faster than U8U8 for low end ARM64 and no difference on accuracy. There is no difference for high end ARM64.
+
+## Transformer-based models
+We have specific optimization for transformer-based models, like QAttention for quantization of attention layer. In order to leverage those specific optimization, you need to optimize your models with [Transformer Model Optimization Tool](https://github.com/microsoft/onnxruntime/tree/master/onnxruntime/python/tools/transformers) before quantizing the model.

```

Comment:
Let's change "we have" to "there are" and in general use more objective wording. There are other areas that use "we" that can also be reworded"